setup.cfg

```diff
@@ -1,2 +1,3 @@
 [flake8]
+ignore = E203, W503
```

Comment:
Can we have specific `# noqa (E203)` comments at each affected line? I'm worried this will cover other cases, like the one in [this link](https://www.flake8rules.com/rules/E203.html) (note that specifying the code in the comment is optional, just to be extra clear about what it's supposed to be ignored, but "noqa" ignores all checks).